ansible-ipi-install/ci/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM quay.io/centos/centos:8
```

Comment:
@smalleni We can keep CI related files in this repo itself and I don't think we need a separate repo for this, we can delete `J-Fleet` repository or use it for some other purpose. any thoughts ? 
We don't need to read any variables from the `env_var` in airflow, all [JSON](https://github.com/mukrishn/airflow-kubernetes/blob/latest_patch/dags/openshift_nightlies/releases/4.7/baremetal/jetski/install.json) inputs are passed to the playbook as `extra_vars` and only few bash script vars are set as `env_var` like keys/tokens.

Reply:
sounds good.